Attributes/Attributes.csproj

```
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
```

```suggestion
        <OutputType>Library</OutputType>
```
Раз атрибуты в отдельном проекте, то этот проект должен быть библиотекой (мы ведь не собираемся запускать проект с атрибутами). Это можно выбрать в настройках проекта.
Тогда main не будет нужен.
(-0.5)